CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- `Label`, `Alert`: improved typing of props ([@qubis741](https://github.com/qubis741)) in [#2212](https://github.com/teamleadercrm/ui/pull/2212))
```

Comment:
We didn't document any type changes before so seems a bit weird to do it for this change

Reply:
so also update of UI lib should be without any changes in changelog?